gammapy/cube/tests/test_simulate.py

```diff
@@ -116,3 +117,19 @@ def test_MDE_sample_background():
     assert_allclose(bkg_evt.table["RA"][0], 265.7253792887848, rtol=1e-5)
     assert_allclose(bkg_evt.table["DEC"][0], -27.727581635186304, rtol=1e-5)
     assert_allclose(bkg_evt.table["MC_ID"][0], 0, rtol=1e-5)
+
+
+@requires_data()
+def test_MDE_sample_psf():
+    psf_map = make_test_psfmap(0.1 * u.deg, shape="gauss")
+
+    dataset = dataset_maker()
+    sampler = MapDatasetEventSampler(random_state=0)
+    src_evt = sampler.sample_sources(dataset=dataset)
```

Comment:
Suggest to call `src_evt` and `src_evt_corr` both `events, like this:
```
events = sampler.sample_sources(dataset=dataset)
events = sampler.sample_psf(psf_map, events)
```
and then below you can have the asserts on `events.table`.

Reply:
Done